authors/apps/articles/serializers.py

```diff
@@ -22,15 +23,57 @@ def update(self, instance, validated_data):
         instance.title = validated_data.get('title', instance.title)
         instance.body = validated_data.get('body', instance.body)
         instance.description = validated_data.get('description', instance.description)
+        if validated_data.get('title'):
+            instance.slug = instance.get_unique_slug()
         instance.save()
         return instance
 
+    
     class Meta:
         model = Articles
-        fields = ('id', 'created_at', 'updated_at', 'author',
-                    'title', 'body', 'description', 'slug')
+        fields = ('id', 'created_at', 'updated_at', 'title',
+        'body', 'description', 'slug', 'average_rating', 'author')
         extra_kwargs = {
             'url': {
                 'view_name': 'articles:article-detail'
             }
+        }
+
+# add ratings serializer
+class RatingsSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    author = serializers.ReadOnlyField(source='rater.username')
+    value = serializers.IntegerField()
+    review = serializers.CharField()
+    article = serializers.ReadOnlyField(source='article.title')
+    slug = serializers.ReadOnlyField(source='article.slug')
+
+    def validate(self, data):
```

Comment:
Make this `validate_value` which will make the `data argument` equivalent to `value`. This will allow for an error message with `value` as the key as opposed to `error` which is currently the case, ie. 
```
{
    "errors": {
        "error": [
            "please keep range of rating from 1-5"
        ]
    }
}
```